tests/functional/test_integration.py

```diff
@@ -1,14 +1,69 @@
 import pytest
+import requests
+import subprocess
+import time
+import yaml
 from ops.model import ActiveStatus
 from pytest_operator.plugin import OpsTest
 
 
 @pytest.mark.abort_on_fail
-async def test_build_and_deploy(ops_test: OpsTest, gunicorn_image, statsd_exporter_image):
+async def test_build_and_deploy_with_psql(
+    ops_test: OpsTest,
+    gunicorn_image,
```

Comment:
We have a standard for type hints, do you think it is applicable here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints